python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -21,7 +21,10 @@
 
 
 def assert_arg_type_is_supported(ty):
-    SUPPORTED = [np.float32, np.float64, np.uint8, np.int8, np.int32, np.int64, np.bool_]
+    SUPPORTED = [
+        np.float32, np.float64, np.uint8, np.int8, np.int32, np.int64,
+        np.bool_, np.float16
```

Comment:
nit: can you move the type to the very beginning to keep all floats together and in increasing order?